src/09-tool-patterns/02-artsy-tools.ts

```diff
@@ -13,11 +13,9 @@ import chalk from "chalk"
 import { CoreMessage, generateText } from "ai"
 import { openai } from "@ai-sdk/openai"
 import dedent from "dedent"
-import {
-  fetchShowsNearLocation,
-  fetchUser,
-  fetchUserArtistFollows,
-} from "../lib/tools"
+import { fetchUser } from "@/tools/fetchUser"
```

Comment:
@anandaroop - This can be for a follow-up, but we don't need to use `@` special characters for absolute imports; we can just set it up so that we use the actual folders. Happy to help set this up!

Reply:
Yep I know it's not a requirement. This is a [Next-ism](https://nextjs.org/docs/app/building-your-application/configuring/absolute-imports-and-module-aliases) I have adopted because it clearly signifies which imports are project-relative absolutes and which are from packages. Without the `@` (or equivalent [sigil](https://en.wikipedia.org/wiki/Sigil_(computer_programming)) — though I like @ because of its mnemonic value for <b><i>a</i></b>bsolute imports) it's needlessly ambiguous. I wish we adopted this more widely! 😉

Reply:
Its more information in the system which = complexity 🤷

Reply:
(There's also no real reason to separate packages from project code, since it doesn't create a problem)